shared/app/nav.desktop.js

```diff
@@ -53,7 +88,14 @@ const mapStateToProps = (state: TypedState) => ({
   _username: state.config.username,
 })
 
-const mapDispatchToProps = (dispatch: Dispatch) => ({})
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  _onHotkey: (cmd: string) => {
+    const tab = hotkeyTabMap[cmd.replace(/(command|ctrl)\+/, '')]
+    if (tab) {
+      dispatch(navigateTo([tab]))
```

Comment:
using `switchTo` would allow the tab stack to persist rather than being blown away each time.

Reply:
TIL 🙌